1228-public_page/1530-files_with_administrative_agreement_is_not_visible.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Files with Administrative Agreement is not visible
+
+    @1530
+    Scenario: Files with Administrative Agreement is not visible
+        Given a User views the Landing Page for a Publication
+        And the Publication contains a File which is an Administrative Agreement
```

Comment:
"And the Publication contains a File that is an Administrative Agreement" (no difference, but (prescriptively) grammatically speaking, you'd need a comma before 'which', thus "And the Publication contains a File, which is an Administrative Agreement"